sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -34,6 +36,28 @@ public DefaultAzureCredentialBuilder authorityHost(String authorityHost) {
     }
 
 
+    /**
+     * Specifies the keep pass database path to read IntelliJ credentials on windows platform. This is required
+     * only on windows platform.
```

Comment:
You might want to make it clear that on other platforms we still support IntelliJ lookup, but we use the standard locations on those platforms.

Reply:
updated.